components/author/context.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from '@wordpress/element';
+
+export const AUTHOR_CONTEXT = createContext();
```

Comment:
This is a bit nit but I feel React Contexts (and any React element) should follow PascalCase nomenclature which is standard on React. It feels odd when it's rendering on other elements

Reply:
I see this pattern being repeated over the PR, if you think it's actionable then it'd be good to normalise!

Reply:
Updated to match the pattern

See:
- [474dba2](https://github.com/10up/block-components/pull/178/commits/474dba26bde6a23aa9d0eab3bea087be76784c6a)
- [437b832](https://github.com/10up/block-components/pull/178/commits/437b832dc4b139b89600dbe08284aa42018e7cff)
- [31e668a](https://github.com/10up/block-components/pull/178/commits/31e668ac27962549d2f68208df1cf738015fbd85)